src/content/docs/iast/troubleshooting/unable-to-detect-vulnerabilities-for-golang-application.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: No vulnerabilities detected for Golang application
+tags:
+  - IAST
+  - New Relic IAST
+  - Troubleshooting
+metaDescription: Troubleshoot Golang application if no vulnerabilities are detected.
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+You don't see any vulnerability for your Golang application.
+
+## Solution
+
+- For Golang, ensure that you imported the required instrumentation packages of libraries and frameworks.
```

Comment:
Why is this a bullet point? We normally don't create a list with just one bullet.